radicle/src/git.rs

```diff
@@ -39,6 +39,13 @@ pub enum ListRefsError {
     InvalidRef(#[from] RefError),
 }
 
+impl From<&RemoteId> for RefString {
+    fn from(id: &RemoteId) -> Self {
+        // PublicKey strings contain only legal characters.
+        RefString::try_from(id.to_string()).unwrap()
+    }
```

Comment:
If you find yourself implementing `Foo::from(some_other_type)`, it's probably better to implement a `From` instance.